src/style/ParagraphProperties.ts

```diff
@@ -16,6 +17,7 @@ const DEFAULT_PAGE_BREAK = false;
 export class ParagraphProperties implements IParagraphProperties {
   private horizontalAlignment: HorizontalAlignment;
   private shouldBreakPageBefore: boolean;
+  private shouldKeepParagraphTogether: boolean;
```

Comment:
Please rename variable to `shouldKeepTogether`. The JSDoc of the method explains what this setting is meant for and I try to name the properties like they are named in the schema (makes it easier in the `toXml method`), if possible.

Reply:
Addressed in commit c107b2f0b4449672eb57777dd8417a35827f13c8